src/Codeception/Configuration.php

```diff
@@ -570,7 +570,22 @@ public static function isEmpty()
      */
     public static function append(array $config = [])
     {
-        return self::$config = self::mergeConfigs(self::$config, $config);
+        self::$config = self::mergeConfigs(self::$config, $config);
+        
+        if (isset(self::$config['paths']['log'])) {
```

Comment:
I think this checks are not needed as this config should be available by default